action.yml

```diff
@@ -34,6 +38,7 @@ runs:
   steps:
       - name: Build Vue
         run: |
+              cd ${{ inputs.path }} || true
```

Comment:
Hi sorry for the stupid question but how does the code in line 41 work and have you tested it?

Reply:
I believe that I had tested it It's also possible that I screwed it up but I'll tell you what:

 I  recommend against the existence of this repository.

 I ended up replacing it was something that I mean it's the whole thing with a lot less work and was not a vue-only solution.

Think it was called build and push. 

when I wake up I might be able to get you a little more detail on what I was trying to achieve with that. I think it was kind of like you can set a width and that with will be path and just like you know object with path who was supposed to enter that directory and build only that directory and then deploy only that directory to github pages instead of assuming content is in the repo root. 

Yes that was the one